src/main/resources/package.json

```diff
@@ -5,6 +5,7 @@
     "bootstrap": "3.1.1",
     "d3": "3.5.17",
     "datamaps": "0.5.8",
+    "handsontable": "0.32.0",
```

Comment:
The latest version is `0.34.0`. Consider updating.

Reply:
changed